src/components/Counter.js

```diff
@@ -1,54 +1,51 @@
-import React, { Component } from "react";
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { increment, decrement } from '../actions';
 
 class Counter extends Component {
-    incrementIfOdd = () => {
-        // Stretch Problem: Implement an increment function that
-        // only increments if the counter value is odd
-    };
+	incrementIfOdd = () => {
+		if (this.props.count % 2 !== 0) {
```

Comment:
Could go for `(this.props.count % 3 == 0)` or `(this.props.count % 3 === 0)` 
Just giving another prospective to go about it.